docs/source/federated-types/composition.mdx

```diff
@@ -184,23 +184,23 @@ If the gateway sent this query to Subgraph A, it would return an error. And with
 
 Composition recognizes this potential issue, and it fails. The hypothetical supergraph schema above would never actually be generated.
 
-`Position.z` is an example of a field in a shared type that is _not_ always resolvable. So now, how do we make sure that such a field _is_ always resolvable?
+`Position.z` is an example of a shared field that is _not_ always resolvable. So now, how do we make sure that such a field _is_ always resolvable?
```

Comment:
Hm, `Position.z` is _shareable_, but I'd argue it isn't currently _shared_, because only one subgraph defines it. I might just stick with "field" here instead of "shared field".

Reply:
Such a good point!